components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -23,6 +25,9 @@ import mozilla.components.lib.crash.R
 class CrashReporterActivity : AppCompatActivity() {
     private val crashReporter: CrashReporter by lazy { CrashReporter.requireInstance }
     private val crash: Crash by lazy { Crash.fromIntent(intent) }
+    private val sendCheckboxKey = "send_checkbox_value"
+    private lateinit var sharedPreferences: SharedPreferences
+    private val sendCheckboxFile = "mozac_lib_crash_settings.xml"
```

Comment:
nit: This can be a lazy property:

```Kotlin
private val sharedPreferences by lazy {
    getSharedPreferences("mozac_lib_crash_settings", Context.MODE_PRIVATE)
}
```

Reply:
Don't I need to add a separate variable for "mozac_lib_crash_settings" file?

Reply:
If it's just in this place then it may be fine.

If you want to you can create a const above the class though:

```Kotlin
private const val SETTINGS_FILE = "mozac_lib_crash_settings"
```